erpnext/stock/get_item_details.py

```diff
@@ -258,14 +258,13 @@ def get_basic_details(args, item, overwrite_warehouse=True):
 		expense_account = get_asset_category_account(fieldname = "fixed_asset_account", item = args.item_code, company= args.company)
 
 	#Set the UOM to the Default Sales UOM or Default Purchase UOM if configured in the Item Master
-	if not args.get('uom'):
```

Comment:
You cannot remove this condition. It will lead to overwriting UOM on server side from https://github.com/frappe/erpnext/blob/version-12-hotfix/erpnext/controllers/accounts_controller.py#L279

I think we should not pass uom from js (https://github.com/frappe/erpnext/blob/version-12-hotfix/erpnext/public/js/controllers/transaction.js#L502). But we need to test it properly, it can lead to other consequences.